### PR TITLE
Update the worker to make it more generic

### PR DESCRIPTION
**Pull request by r0x0d**
The idea behind this commit is to make the worker even more generic, not only being able to execute bash scripts, but, let the user decide which type of script it will be executed, by defining a `interpreter` key in the partial yaml playbook, which will tell the worker what to execute.

Changes included in this commit:

* Rename every reference from rhc-worker-bash to rhc-worker
* Added the `interpreter` key to be parsed in the incoming playbook 
** Use the `interpreter` key instead of hardcoding the /bin/bash path
    for the command execution
* Updated the references in the project to build using the new name

### CODE REVIEW

File: README.md
Comment (andywaltlova) [inline README.md:7]: Maybe something like this? Also depending on the title change to outline below needed

```suggestion
# RHC Worker

Remote Host Configuration (rhc) worker for executing  scripts on hosts
managed by Red Hat Insights. Interpreter used to execute the script is defined inside the supplied yaml file - served by insights.
```

File: .packit.yaml
Comment (andywaltlova) [inline .packit.yaml:13]: Is it because of this that the copr build is not runned on PR? I mean because of the repo still has old name

**Review by andywaltlova — CHANGES_REQUESTED** [review]
I am also thinking if the `rhc-worker` name is okay from insights POV, are they okay with it? I would expect something like `rhc-worker-multipurpose` or `rhc-worker-generic` because of the existence of `rhc-worker-playbook`, I mean .. the specific name just implies better what is the purpose of the package I think.

Apart from the few questions, the changes looks good to me!

File: .packit.yaml
Comment (r0x0d) [reply .packit.yaml:13]: Yes, exactly!

**Comment by andywaltlova** [comment]
Alright `rhc-worker-script` sounds reasonable

File: README.md
Comment (andywaltlova) [inline README.md:9]:
```suggestion
- [RHC Worker Script](#rhc-worker-script)
```

**Review by andywaltlova — CHANGES_REQUESTED** [review]
- `server.go`  docstring for `Send` method needs update (still mentions executing bash script
- `runner.go` L92 docstring as well 
- CONTRIBUTING file also mentions bash
- Summary in specfile still mentions bash

**Comment by andywaltlova** [comment]
No builds for https://copr.fedorainfracloud.org/coprs/g/oamg/rhc-worker-script/builds/ yet, I don't know enough about github actions to know if it will be triggered in this PR or only after we merge it - I don't think it should block the merge :)

**Review by andywaltlova — APPROVED** [review]
Looks good now!

**Pull request merged**
